src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -91,14 +90,14 @@ public ResponseEntity<Integer> getContingencyCount(@Parameter(description = "Con
         return ResponseEntity.ok().body(service.getContingencyCount(ids, networkUuid, variantId));
     }
 
-    @GetMapping(value = "/contingency-lists/{id}/export", produces = MediaType.APPLICATION_JSON_VALUE)
+    @GetMapping(value = "/contingency-lists/export", produces = MediaType.APPLICATION_JSON_VALUE)
     @Operation(summary = "Evaluate and export a contingency list to PowSyBl JSON format")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The contingency list in PowSyBl JSON format"),
                            @ApiResponse(responseCode = "404", description = "The contingency list does not exists")})
-    public ResponseEntity<List<Contingency>> exportContingencyList(@PathVariable("id") UUID id,
-                                                                   @RequestParam(value = "networkUuid", required = false) UUID networkUuid,
-                                                                   @RequestParam(value = "variantId", required = false) String variantId) {
-        return ResponseEntity.ok().body(service.exportContingencyList(id, networkUuid, variantId));
+    public ResponseEntity<Contingencies> exportContingencyList(@RequestParam(value = "networkUuid", required = false) UUID networkUuid,
```

Comment:
Do we really want to accept callings without `networkUuid` value ?

Reply:
good remark , fixed

Reply:
i reverted this change , because 2 tests fails, they call this endpoint without `networkUuid`